### DISCUSSION TITLE
What are your hobbies / creative outlets outside of coding?

### DISCUSSION THREAD

**Discussion by filipkappa** [mdbootstrap/TW-Elements#1401] [Hang out & have fun]
We all know that there's no sound more pleasant than the hum of your laptop fan, and no better feeling than that click in your head when you realized you just solved something.

But what outside of that?

Do you have any way to express yourself creatively, unwind, and recharge after a long day of coding?

**Comment by filipkappa** [comment]
I'll start. I love to play tabletop RPGs.

I played it a bit when I was a child & rediscovered it in recent years.
I'm currently participating in one campaign and DMing (running) another one.

In my opinion, it's one of the best hobbies to "escape" reality for a bit.

For approximately 4-6 hours every couple of weeks, you get to be a dwarven warrior that doesn't care about your everyday struggles & stress, because he has a much more important task at hand (usually saving the world). 

Playing D&D is currently one of the top things I do for fun, and running my own game allows me to warm up my creativity.

**Comment by cyggarro** [comment]
When I'm not coding with Tailwind, I enjoy reading thrillers by Sebastian Fitzek. I spent most of my days reading them now.
I also got my first shot at truly remote work last year, so I'm trying to work from different places & get to know the cities I'm staying in.
Greetings from Prague right now! 😃

**Reply by filipkappa** [reply]
Awesome to hear that! What's your next destination? 🗺️

**Comment by Shigure92** [comment]
I have an enormous amount of hobbies but creative wise I engage in two types of activities. I draw and/or paint. Not very often because there is a point where you run out of wall space in your home to hang things.

I also sew. Clothes, home decor pieces, and accessories. Every time I do that I also explore the creativity of my cat. When I get back to my room I get to find out what kind of mess he was able to do this time.

**Reply by filipkappa** [reply]
Do you like to draw/paint specific things?

**Reply by Shigure92** [reply]
I paint mostly people, but currently, I'm learning urban sketching. My GitHub avatar is one of my drawings :)

**Comment by juujisai** [comment]
In my free time I like to play video games, chess or just listen to music. Over the last few years I've been listening to piano pieces a lot and it makes me want to learn to play at least a little bit in the near future.

Warm weather is near, so it will be time for one of my newest hobbies - camping. I'm hoping for warm weather so I can just chill and spend some time outside.

**Reply by filipkappa** [reply]
Do you mean classical piano, or something else?
Are there any pieces you would recommend?

**Reply by juujisai** [reply]
Yes, classical piano. I used to listen to Chopin a lot, but right now my absolute favorite is Debussy's Clair De Lune.

**Reply by filipkappa** [reply]
There is a spell in DnD called "feather fall". It makes you fall slower from high distances.

I always play Clair De Lune in the background when somebody is using this spell because it paints a majestic image.

**Comment by smolenski-mikolaj** [comment]
I love to play football and do all kinds of active recreation, such as cycling, sailing or skiing. Sometimes I also play guitar (shanty of course) or tinker with Arduino. In addition, recently I have been testing the limit of my endurance by taking care of my two sons.

**Reply by filipkappa** [reply]
Cycling, skiing & raising children.

Seems like a hardcore triathlon 🤣

**Comment by Trochonovitz** [comment]
Typical: books (non-fiction > fiction) and video games (here without distinction). Recently LEGO but still waiting for bigger condo. For 2023 I have resolution to sign in for additional french lessons... But everyone know how these resolutions working. :>

**Reply by filipkappa** [reply]
> Recently LEGO but still waiting for a bigger condo.

Do you mean that you have so many LEGOs that they don't fit in your apartment? :O

**Reply by Trochonovitz** [reply]
Ha ha I wish I had. I want to my sets proper exposition not storage them in some bad looking cabinets.

**Comment by ascensus-mdb** [comment]
I used to be a proffesional dancer 🕺

Now I don't do it professionally, but dance is still a big thing for me

https://youtu.be/bbKBlGxwgj8

**Comment by bwsky-a** [comment]
In my free time, apart from coding, I like playing video games (RPG  and open-world), but the most interesting for me are games related to Star Wars because I'm a fan of the trilogy and I like the fact that you can play as a Jedi knight and do various missions :)

Apart from that, I also love to play football and travel. For me the best are trips to less touristy places (or off-season) then you can get to know a completely different culture and customs than the one prepared for tourists

**Comment by mako3577** [comment]
A huge fan of football ⚽ I follow news, watch the matches and also play myself. As a child, I even belonged to a local club.

Cooking is my another hobby. My best specialities are ramen, pho and pizza. In my opinion, the best and most enjoyable way to get culinary inspiration is traveling, and this is one of the reasons why I visit foreign countries.

**Comment by TheoIcelander** [comment]
I'm spending most of my free time outside.
I love hiking.

Where I live we also do something called glacier walks - basically exploring the glaciers, caves & other wonders of nature in Iceland!

**Comment by JohnGregorySpook** [comment]
I'm an avid reader of fantasy and science fiction.
